### ISSUE TITLE
Suggestion: use `const ipfs = new IPFS` over `const node = new IPFS`

### ISSUE THREAD

**Issue by alanshaw** [ipfs-inactive/old-js-ipfs-website#236]
`node` might be a bit confusing with Node.js or DAG node.

**Comment by daviddias** [comment]
🚲 🏡 

😁

**Comment by alanshaw** [comment]
> 🚲 🏡

Point taken, but not just my opinion 😆!

I opened this issue as I had a session watching @terichadbourne get up to speed with the files API and noticed this as a stumbling block. I thought that a rename might help, but appreciate this is probably subjective. Personally I'm ok with it either way but I'd like to make onboarding easier. @terichadbourne please say if my observation was incorrect or if you have any alternatives.

**Comment by terichadbourne** [comment]
@alanshaw If I'm remembering correctly, you probably took this note while I was looking at the underlying code that creates the "Exchange Files in Browser" example at https://js.ipfs.io/ and I was guessing that `node` is what you call a new instance of `ipfs`. 

At the time, one of the reasons that seemed odd to me is that I had previously only worked with the DAG API, which I learned through the lessons on ProtoSchool, and there you see the format `ipfs.dag.whatever()`. I believe you mentioned to me that there was just a change made so that you now use `node.whatever()` instead of `node.file.whatever()` when working with files. 

I just went and looked at the docs for both the DAG and file APIs. The [DAG docs](https://github.com/ipfs/interface-ipfs-core/blob/master/SPEC/DAG.md) use `ipfs.dag.whatever()` and the [file docs](https://github.com/ipfs/interface-ipfs-core/blob/master/SPEC/FILES.md) uses `ipfs.whatever()`, NOT `node.whatever()` as shown in that specific example.  

Am I right to understand that it's the user choosing whether they say `node` or `ipfs` or `zebra` here, and they just have to be consistent within their own code? If so, I do think we should be consistent between our examples and our documentation, and personally it would make more sense to me as IPFS, but I say this as someone who doesn't know a ton about the technical definition of nodes. I agree that there's more likelihood for confusion with `node` than `IPFS`, although when I saw it I did assume that the meaning of node was like a node in a graph or tree, and that that must be what you call one unit of IPFS that you make and add stuff to.
